src/base/QXmppJingleIq.cpp

```diff
@@ -839,37 +851,76 @@ void QXmppJingleIq::setResponder(const QString &responder)
     d->responder = responder;
 }
 
-/// Returns true if the call is ringing.
-
-bool QXmppJingleIq::ringing() const
+/// Returns the session ID.
+QString QXmppJingleIq::sid() const
 {
-    return d->ringing;
+    return d->sid;
 }
 
-/// Set to true if the call is ringing.
+/// Sets the session ID.
 ///
-/// \param ringing
+/// \param sid
 
-void QXmppJingleIq::setRinging(bool ringing)
+void QXmppJingleIq::setSid(const QString &sid)
 {
-    d->ringing = ringing;
+    d->sid = sid;
 }
 
-/// Returns the session ID.
+///
+/// Returns the state of an RTP session as specified by \xep{0167, Jingle RTP Sessions}
+/// Informational Messages.
+///
+/// \return the session's state
+///
+/// \since QXmpp 1.5
+///
+QXmppJingleIq::RtpSessionState QXmppJingleIq::rtpSessionState() const
+{
+    return d->rtpSessionState;
+}
```

Comment:
doesn't seem to be tested

Reply:
Yes, thus the pull request is in Draft state and the corresponding checkbox unchecked.